### DISCUSSION TITLE
Part list for upgrading from V4 One slot to seven slot

### DISCUSSION THREAD

**Discussion by RottenToTheGore** [sanni/cartreader#488] [Need help]
Was able to buy a V4 one slot kit from someone, but I'm torn on if I want to have a full seven slot instead of swapping adapters when needed (which is rare anyways, but...options). So besides the seven slot PCB, and a different 3d printed frame, what would I need? I guess some of the different screws/risers and such?

Also curious if there's any difference between the two configurations. Besides weight and height I can't see anything.

Thanks!

**Comment by sanni** [comment]
The seven slot upgrade for V4 never really worked that well. It's the reason why I made V5. Exchanging your V4 PCB for a V5 is the option that will give you the least headache.

**Reply by RottenToTheGore** [reply]
Appreciate the response! I think I'll stick with what I have now and get the other PCBs later maybe.

Also, I can't seem to find the picture about building the V4 without the battery module. Could you help me find it?

Thanks!

**Reply by sanni** [reply]
![wSujVBeU](https://user-images.githubusercontent.com/19169494/186170275-c4070848-2368-4c90-9142-cd6d4b8ed387.jpeg)
![68g1Pxo4](https://user-images.githubusercontent.com/19169494/186170283-dcc33bc6-e1ba-4db5-8bb0-3a9cd3f95a93.jpeg)
